servers/src/test/java/tachyon/master/next/filesystem/meta/InodeDirectoryTests.java

```diff
@@ -104,9 +104,11 @@ public void removeNonExistentChildTest() {
   @Test
   public void deleteInodeTest() {
     InodeDirectory inode1 = createInodeDirectory();
-    Assert.assertEquals(false, inode1.isDeleted());
+    Assert.assertFalse(inode1.isDeleted());
     inode1.delete();
-    Assert.assertEquals(true, inode1.isDeleted());
+    Assert.assertTrue(inode1.isDeleted());
+    inode1.restore();
```

Comment:
Was restore() added? The changes in this file seem unrelated to the traversal change.